### DISCUSSION TITLE
Visualize setInterval

### DISCUSSION THREAD

**Discussion by arturcarvalho** [statelyai/xstate#2650] [Q&A] [answered]
Hi there,

i'm trying to visualize the timerMachine described here: https://github.com/statelyai/xstate/blob/main/docs/tutorials/7guis/timer.md

But I can't make it work on stately.ai or xstate.js.org/viz. On stately.ai, it complains about "setInterval" not defined. This is my attempt: https://stately.ai/viz/68224db7-1a34-4338-aaf7-f909308c8ec1

Thanks and have a nice day!
Artur

**Comment by davidkpiano** [comment, marked as answer]
Reported here: https://github.com/statelyai/xstate-viz/issues/205

**Reply by arturcarvalho** [reply]
Great, thanks! I'll wait for the next version bump.
